servlet-map/src/main/resources/META-INF/resources/spring-map-jsp/index.jsp

```diff
@@ -121,6 +121,7 @@
             <%-- If logout url is present - so logout link --%>
             <c:when test="${!empty _logout_uri}">
                 <a href="${pageContext.request.contextPath}${_logout_uri}"><spring:message code="logout" text="Logout" /></a>
+                <a href="javascript:void(0);" id="oskari-profile-link"><spring:message code="account" text="Account" /></a>
```

Comment:
Maybe use href="${pageContext.request.contextPath}${_registration_uri}" as it works as the user modify page. That way it works properly even if personaldata bundle is not loaded.